chart/values.yaml

```diff
@@ -97,10 +97,17 @@ media:
       subPath: media
 
 contentdatabase:
-  provider:
-    tmdb:
-      apikeyKey: tmdb_apikey
-      existingSecret: bigsecret
+  # TheMovieDB
+  tmdb:
+    enabled: true
+    apikeyKey: tmdb_apikey
+    existingSecret: bigsecret
+  # TVDatabase
+  tvdb:
+    enabled: true
+    apikeyKey: tvdb_apikey
+    pinKey: tvdb_pin
+    existingSecret: bigsecret
```

Comment:
i think you would need two exitstingSecrets (one for api key & one for pin key, right?)

Reply:
They are both expected to come from the same existingSecret.  All references at same level/place of the map use the same existing secret.  That secret just has multiple keys.  Chart readme includes an example that just uses a single secret for specifying all the key/value pairs.

Resolving this conversation.  Please reopen if you have any additional questions on this particular part.